toast_planck/reproc_ring.py

```diff
@@ -1955,6 +2006,20 @@ def build_templates(self, rings):
             self.mapsampler_freq = mapsampler_fg
             self.mapsampler_freq_has_dipole = False
 
+        # Load FSL beam mask and corresponding low resolution sky model (one for each detector)
+        if self.fslbeam_mask_path is not None:
+            if self.fslbeam_mask == None:
```

Comment:
Comparison to `None` should be done with `is`, not `==`. This used to generate a warning, maybe it still does.

Reply:
Fixed.